tests/providers/amazon/aws/operators/test_rds.py

```diff
@@ -450,3 +452,69 @@ def test_delete_event_subscription(self):
 
         with pytest.raises(self.hook.conn.exceptions.ClientError):
             self.hook.conn.describe_event_subscriptions(SubscriptionName=EXPORT_TASK_NAME)
+
+
+@pytest.mark.skipif(mock_rds is None, reason='mock_rds package not present')
+class TestRdsCreateDbInstanceOperator:
+    @classmethod
+    def setup_class(cls):
+        cls.dag = DAG('test_dag', default_args={'owner': 'airflow', 'start_date': DEFAULT_DATE})
+        cls.hook = RdsHook(aws_conn_id=AWS_CONN, region_name='us-east-1')
+
+    @classmethod
+    def teardown_class(cls):
+        del cls.dag
+        del cls.hook
+
+    @mock_rds
+    def test_create_db_instance(self):
+        create_db_instance_operator = RdsCreateDbInstanceOperator(
+            task_id='test_create_db_instance',
+            db_instance_identifier=DB_INSTANCE_NAME,
+            db_instance_class="db.m5.large",
+            engine="postgres",
+            rds_kwargs={
+                "DBName": DB_INSTANCE_NAME,
+            },
```

Comment:
If we have `db_instance_identifier` why do we need `DBName` in `rds_kwargs`?

Reply:
In case with Postgres it's different parameters - `db_instance_identifier` identifies the DB instance, while `DBName` is the name of the database to create when the DB instance is created.

We could omit `DBName`, then a database named `postgres` is created in the DB instance, but I think it'd be better to show how to use `rds_kwargs` in the example DAG.